app/api/events.py

```diff
@@ -128,19 +183,29 @@ def after_create_object(self, event, data, view_kwargs):
         uer = UsersEventsRoles(user, event, role)
         save_to_db(uer, 'Event Saved')
 
-    decorators = (accessible_events, )
+    def before_create_object(self, data, view_kwargs):
+        data['starts-at'] = data['starts-at'].replace(tzinfo=None)
+        data['ends-at'] = data['ends-at'].replace(tzinfo=None)
+
+    def after_get(self, result):
+        result = changeToTimezone(result)
+
+    def after_post(self, result):
+        result = changeToTimezone(result)
+
+    decorators = (jwt_required,)
     schema = EventSchema
     data_layer = {'session': db.session,
                   'model': Event,
-                  'methods': {
-                    'query': query,
-                    'after_create_object': after_create_object
-                  }}
+                  'methods': {'before_create_object': before_create_object}
```

Comment:
You removed call to ```query``` and ```after_create_object``` method here. Please add it here.
see  
https://github.com/fossasia/open-event-orga-server/pull/3726/files#diff-364bbff4749f94ed8f7082208c36cf78R137